cli/tests/integrations/helpers/common.py

```diff
@@ -21,10 +21,10 @@ def exec_command(cmd, env=None, stdin=None, timeout=None):
     :type env: dict | None
     :param stdin: File to use for stdin
     :type stdin: file
-    :param timeout: If the process doesn't terminate after this timeout,
-                    it will get killed and a subprocess.TimeoutExpired
-                    exception will be raised. The timeout is in seconds.
+    :param timeout: The timeout for the process to terminate.
     :type timeout: int
+    :raises: subprocess.TimeoutExpired when the timeout is reached
```

Comment:
Is `raises` a field understood by pydoc? Right now we don't run pytdoc over the CLI code, but I'd like to start doing this again in the future.

Reply:
It seems like it's understood by sphinx, I'll make some more research for pydoc. I have seen this somewhere else in the codebase though.

Reply:
I don't think the syntax we currently use is specific to pydoc, according to https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format#answer-24385103, it looks more similar to reST (though I couldn't find specification for pydoc). Could you confirm?